exonum/src/api/error.rs

```diff
@@ -57,6 +58,44 @@ pub enum Error {
     Unauthorized,
 }
 
+/// A helper structure allowing to build `MovedPermanently` response from the
+/// composite parts.
+#[derive(Debug)]
+pub struct MovedPermanentlyErrorBuilder {
```

Comment:
Nit: It could make sense to refactor this type as just `MovedPermantentlyError`, with a conversion into `api::Error`, since the error is ready once constructed.